py/openage/pack/pack_config.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2015-2015 the openage authors. See copying.md for legal info.
+
+import configparser
+import re
+
+
+class PackConfig:
+    """
+    main config file for a mod pack.
+
+    stores the package metainformation for the mod list.
+
+    file contents:
+
+    [openage-pack]
+    name = <package name>
+    version = <package version>
```

Comment:
This was intended to be the revision of the content. But yeah, the name is ambiguous.